guides/docs-Red_Hat_Satellite_6/common/modules/proc_installing-postgresql.adoc

```diff
@@ -0,0 +1,114 @@
+[id="installing-postgresql_{context}"]
```

Comment:
This directory naming is odd. We don't have such a guide in upstream.

Reply:
Removed